packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -708,17 +708,28 @@ export class GarbageCollector implements IGarbageCollector {
 			return;
 		}
 
+		// If sweep is disabled, we'll tombstone both tombstone-ready and sweep-ready nodes.
+		const { toTombstone, toDelete } = this.configs.shouldRunSweep
+			? {
+					toTombstone: [...tombstoneReadyNodes],
```

Comment:
Should we make the param to this function readonly sets so that these are not modified?

Reply:
That's not supported.  I found this article, I don't think it's worth pursuing anything here (didn't give it a ton of thought honestly)

https://www.techiedelight.com/create-immutable-set-javascript/